src/hastings_vacuum.erl

```diff
@@ -34,13 +34,18 @@
 
 -export([
     handle_db_event/3,
-    clean_db/1
+    clean_db/1,
+    clean_db/2,
+    clean_sharddb/1,
```

Comment:
`clean_sharddb` is a bit difficult to read. Can you change it to `clean_shard_db`?